dds/DCPS/LinuxNetworkConfigMonitor.cpp

```diff
@@ -78,12 +78,14 @@ bool LinuxNetworkConfigMonitor::open_i()
   if (socket_.open(addr, AF_NETLINK, NETLINK_ROUTE) != 0) {
 #ifdef ACE_ANDROID
     if (log_level >= LogLevel::Warning) {
-      ACE_ERROR((LM_WARNING, "(%P|%t) WARNING: LinuxNetworkConfigMonitor::open_i: could not open socket"
-                 " this is expected for API30+\n"));
+      ACE_ERROR((LM_WARNING, "(%P|%t) WARNING: LinuxNetworkConfigMonitor::open_i: "
+        "could not open Netlink socket "
+        "(this is expected for API>=30, see Android section in the Developer's Guide)\n"));
     }
```

Comment:
```suggestion
        "(this is expected for API>=30, see Android section in the Developer's Guide)\n"));
      return false;
    }
```
Should this skip logging the ERROR below?

Reply:
There's already a `return false` after the `#endif`.

Reply:
Right, I missed that...